torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4547,6 +4547,16 @@ def _tensor(shape, dtype=dtype, low=None, high=None):
 
     return [SampleInput(tensor, args=args) for tensor, args in test_cases]
 
+def sample_inputs_one_hot(op_info, device, dtype, requires_grad, **kwargs):
+    def make_tensor_(shape, *, low, high):
+        return make_tensor(shape, device=device, dtype=dtype, low=low, high=high, requires_grad=requires_grad)
+
+    return [
```

Comment:
Could we test a scalar tensor input? e.g. `SampleInput(make_tensor([], low=0, high=10))` as well as `SampleInput(make_tensor([], low=0, high=5), kwargs=dict(num_classes=10))`